.github/workflows/dashboards_cypress_workflow.yml

```diff
@@ -0,0 +1,143 @@
+name: Orchestrator cypress workflow
+run-name: dashboards_cypress_workflow ${{ inputs.UNIQUE_ID != '' && inputs.UNIQUE_ID || '' }} # Unique id number appended to the workflow run-name to reference the run within the orchestrator.
```

Comment:
Is a better name for this file perhaps `release_cypress_workflow` since it will execute the tests within this repo against a release artifact.